src/main/java/seedu/address/logic/commands/AppendLogCommand.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Appends the results of a FindCommand to the logbook.
+ */
+public class AppendLogCommand extends Command {
+
+    public static final String COMMAND_WORD = "alog";
+
+    public static final String COMMAND_WORD_ALIAS = "al";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "or" + COMMAND_WORD_ALIAS
+            + ": Appends the results of the most recent FindCommand to the logger tab.\n"
+            + "Example: " + COMMAND_WORD + "or" + COMMAND_WORD_ALIAS;
```

Comment:
I think would be good to split into 2 examples, it is to standardise with the format in other classes.
```suggestion
            + "Example 1: " + COMMAND_WORD + "\n";
            + "Example 2: " + COMMAND_WORD_ALIAS;
```